frontends/api/src/mitxonline/clients.ts

```diff
@@ -24,29 +24,37 @@ const BASE_PATH =
 
 const usersApi = new UsersApi(undefined, BASE_PATH, axiosInstance)
 const b2bApi = new B2bApi(undefined, BASE_PATH, axiosInstance)
-const enrollmentsApi = new EnrollmentsApi(undefined, BASE_PATH, axiosInstance)
 const programsApi = new ProgramsApi(undefined, BASE_PATH, axiosInstance)
 const programCollectionsApi = new ProgramCollectionsApi(
   undefined,
   BASE_PATH,
   axiosInstance,
 )
+
 const programCertificatesApi = new ProgramCertificatesApi(
   undefined,
   BASE_PATH,
   axiosInstance,
 )
+
 const coursesApi = new CoursesApi(undefined, BASE_PATH, axiosInstance)
+
 const courseCertificatesApi = new CourseCertificatesApi(
   undefined,
   BASE_PATH,
   axiosInstance,
 )
 
+const courseRunEnrollmentsApi = new EnrollmentsApi(
+  undefined,
+  BASE_PATH,
+  axiosInstance,
+)
+
 export {
   usersApi,
   b2bApi,
-  enrollmentsApi,
+  courseRunEnrollmentsApi,
```

Comment:
This is kind of a side effect of some work I had in progress on this branch for adding the program enrollments API, just renames the `enrollmentsApi` to `courseRunEnrollmentsApi`, because that's what it is.